### ISSUE TITLE
signup for people going to each game

### ISSUE THREAD

**Issue by cbolson1** [tommybananas/mufa-info-api#3]
Every year my team has to come up with a method for people to say if they are going to a particular game or not.  We usually revert to something like Google docs for that purpose.  I'm just curious if you have thought about integrating that feature into this app.  I realize it would involve a lot of overhead to do it securely, but there may be an insecure way to do it easily

**Comment by tommybananas** [comment]
I would love to do stuff like that. I also wanted to give special access to captains and such. The main hurdle is that all my attempts to integrate with the org have failed, so we'd have to do manual account vetting of some kind (probably not a huge deal). Also a public message board with game/team/field filters would allow everyone to communicate quite easily, facebook login would be an easy way to create identity system.

**Comment by cbolson1** [comment]
Yeah, I don't think you can rely upon integrating with the org.  All you can do is use the publicly available information and present it in a fashion that is way better than MUFA does.  That's the best thing you have going for this app right now.

**Comment by cbolson1** [comment]
That being said, do you think you need some form of an authentication system?  I'm wondering if we can just make the signup completely open for everyone to read and write.  We would just need 1 signup board per team per game.  We should probably split that into Men and Women.

**Comment by cbolson1** [comment]
Do you think having a wide open signup sheet would lead to people abusing the system (e.g. messing with other teams signups)?

**Comment by tommybananas** [comment]
I think the easiest option is to just create a "message board" for each layer: league, team, game and allow write access to everyone via facebook authentication (everyone has facebook right?). That way we get profile pictures and stuff for free and it mitigates spam at least a little bit. It also allows people to subscribe to "updates" at whatever layer they like. ie league board would house weather cancelations and league news, game/team board would be for attendance, etc.

**Comment by cbolson1** [comment]
I like the idea of the message board for each league and team, but I would like a little more structure for the game board (simple yes/no/maybe dropdown option for each player on the team)

**Comment by cbolson1** [comment]
Also, yes, everyone has Facebook.  From a user perspective though, one thing that's nice about the app is you don't have to login.  It's super quick to access the info right now.

**Comment by tommybananas** [comment]
Well I figure if people are taking the time to write a post they should be
able to login. It's one screen to login and it will automatically be cached
for future. I'm totally fine with the extended options for game attendance
and such though.. realistically I probably won't have time to help that
much so anything you want to work on I'm happy to help advise and give you
access to mobile project as well

**Comment by cbolson1** [comment]
Totally agree.  People "should" be able to login if they want to write a post.  I was simply trying to say that we should keep the barrier to entry as low as possible.  In general, I feel the more hurdles that people have to jump through in order to use your app, the fewer users you will have.

**Comment by cbolson1** [comment]
That being said, I won't have a ton of time to work on this either.  However, I'd be happy to take a look at some of these features when I do have time.  Please do share the mobile project as well, and I'll see what I can do

**Comment by tommybananas** [comment]
Sounds good, if spam gets bad we can require fb login
